backend/de.metas.acct.base/src/main/java/de/metas/acct/gljournal_sap/service/SAPGLJournalRepository.java

```diff
@@ -70,10 +70,10 @@ public SAPGLJournal create(
 		return loaderAndSaver.create(createRequest, currencyConverter);
 	}
 
-	public void setDocStatus(final @NonNull SAPGLJournalId glJournalId, final String code)
+	public void setDocStatus(final @NonNull SAPGLJournalId glJournalId, final String docStatus)
 	{
 		final I_SAP_GLJournal sourceRecord = InterfaceWrapperHelper.load(glJournalId, I_SAP_GLJournal.class);
-		sourceRecord.setDocStatus(DocStatus.Reversed.getCode());
+		sourceRecord.setDocStatus(docStatus);
 		InterfaceWrapperHelper.save(sourceRecord);
 	}
```

Comment:
like mentioned above, pls drop the `SAPGLJournalRepository.setDocStatus` method and reuse `SAPGLRepository.updateById`

Reply:
done